dashboard/src/components/AppView/AccessURLTable/AccessURLTable.test.tsx

```diff
@@ -4,15 +4,32 @@ import * as React from "react";
 
 import itBehavesLike from "../../../shared/specs";
 
+import ResourceRef from "shared/ResourceRef";
 import LoadingWrapper from "../../../components/LoadingWrapper";
 import { IIngressSpec, IResource, IServiceSpec, IServiceStatus } from "../../../shared/types";
 import AccessURLItem from "./AccessURLItem";
 import AccessURLTable from "./AccessURLTable";
 
-describe("componentDidMount", () => {
-  it("fetches ingresses", () => {
+const defaultProps = {
+  services: [],
+  ingresses: [],
+  ingressRefs: [],
+  getResource: jest.fn(),
+};
+
+describe("when receiving ingresses", () => {
+  it("fetches ingresses at mount time", () => {
+    const mock = jest.fn();
+    shallow(
+      <AccessURLTable {...defaultProps} ingressRefs={[{} as ResourceRef]} getResource={mock} />,
+    );
+    expect(mock).toHaveBeenCalled();
```

Comment:
Shouldn't this be `.toHaveBeenCalledWith(...)` to be able to justify the name of the test? (ie. to show that it fetches ingresses at mount time)